ci/pipeline.yml

```diff
@@ -386,78 +386,94 @@ jobs:
-        - task: nightly-builds
-          config:
-            inputs: [name: src]
-            <<: *cf-image
-            run:
-              path: src/ci/tasks/run-task.sh
+          CF_APP_NAME: federalist-queues-ui-staging
+          CF_MANIFEST: src-queues-ui/.cloudgov/manifest.yml
+          CF_VARS_FILE: src-queues-ui/.cloudgov/vars/staging.yml
+          CF_PATH: src-queues-ui
+        on_failure:
+          try:
+            task: cancel-queues-ui-deployment
+            config:
+              <<: *cf-image
+              inputs: [name: src-queues-ui]
+              run:
+                path: src-queues-ui/ci/tasks/cancel-deployment.sh
+            params:
+              <<: *staging-cf
+              CF_APP_NAME: federalist-queues-ui-staging
+    on_failure:
+      in_parallel:
+        - put: gh-status
+          inputs: [src-queues-ui]
+          params: {state: failure}  
+        - put: slack
           params:
-            <<: *staging-cf
-            CF_APP_NAME: federalistapp-staging
-            CF_TASK_NAME: nightly-builds
-            CF_TASK_COMMAND: yarn nightly-builds
-        - task: remove-inactive-federalist-users
-          config:
-            inputs: [name: src]
-            <<: *cf-image
-            run:
-              path: src/ci/tasks/run-task.sh
+            text: |
+              :x: FAILED: pages queues UI deployment on staging
+              ((build-url))
+              <@U01J8B463T5> <@U01J0M3E8NS> <@U01J7LKV5MZ>
+            channel: ((slack-channel))
+            username: ((slack-username))
+            icon_url: ((slack-icon-url))
+    on_success:
+      in_parallel:
+        - put: gh-status
+          inputs: [src-queues-ui]
+          params: {state: success}
+        - put: slack
           params:
-            <<: *staging-cf
-            CF_APP_NAME: federalistapp-staging
-            CF_TASK_NAME: remove-inactive-federalist-users
-            CF_TASK_COMMAND: yarn remove-inactive-federalist-users
+            text: |
+              :white_check_mark: SUCCESS: Successfully deployed pages queues UI on staging
+              ((build-url))
+              <@U01J8B463T5> <@U01J0M3E8NS> <@U01J7LKV5MZ>
+            channel: ((slack-channel))
+            username: ((slack-username))
+            icon_url: ((slack-icon-url))
 
-  - name: every-ten-minutes-staging
+  - name: nightly-tasks-staging
     plan:
       - get: src
         resource: src-staging
         params: {depth: 1}
-      - get: every-10m
+      - get: nightly
         trigger: true
-      - task: timeout-builds
+      - task: restage-api
         config:
           inputs: [name: src]
           <<: *cf-image
           run:
-            path: src/ci/tasks/run-task.sh
+            path: src/ci/tasks/restage.sh
         params:
           <<: *staging-cf
           CF_APP_NAME: federalistapp-staging
-          CF_TASK_NAME: timeout-builds
-          CF_TASK_COMMAND: yarn timeout-builds
-
-
+      - in_parallel:
```

Comment:
the `in_parallel` isn't doing anything since only one task is inside of it. We should either remove it or move it above both tasks to enable them to run in parallel.

Reply:
thanks @davemcorwin  ... changed to restage apps in parallel

Reply:
thanks @amirbey !